alexberka/a-trivial-glance
# Player Game Selection Page

## Description
Finished refactoring of /game from MVP to /games and /game/[id] for game-specific player display
Removed /game.js and /host/game.js paths

Users clicking 'JOIN' from welcome page are taken to a page showing all the live games to choose from. Clicking on one will take user to the user game display page for that game.

Nav menu 'Switch to Host/Player View' redirects to opposite view of specific game user is viewing, if that game is live ('if moving to player view') or owned by user ('if directing to host view'); else, directs to '(/host)/games'

Added 'HOST' tag under nav menu for all host views

Added user info around 'Sign Out' button on Welcome page

Styling adjustments

## Related Issue
#22 

## Motivation and Context
As a player, I want to be able to select the game that is being played at the establishment I am in, and only see questions from that game.

## How Can This Be Tested?
Select 'JOIN' from welcome page. All open games from all users should be visible. Click on a game to be taken to the game display page, where the current open question and all 'released' questions are shown. Page updates every two seconds, so if the game is closed, player will redirect.

## Screenshots (if appropriate):

## Types of changes
